index.html

```diff
@@ -28,14 +28,14 @@ <h2>I’am a Software Developer</h2>
              don’t hestiate to contact me.
             </p>
             <ul>
-                <li><img src="images/Vector-1.png" alt=""></li>
-                <li><img src="images/Group.png" alt=""></li>
-                <li><img src="images/hand.png" alt=""></li>
-                <li><img src="images/twitter.png" alt=""></li>
-                <li><img src="images/microverse.png" alt=""></li>
+                <li><a href=""><img src="images/Vector-1.png" alt=""></a></li>
+                <li><a href=""><img src="images/Group.png" alt=""></a></li>
+                <li><a href=""><img src="images/hand.png" alt=""></a></li>
+                <li><a href=""><img src="images/twitter.png" alt=""></a></li>
+                <li><a href=""><img src="images/microverse.png" alt=""></a></li>
             </ul>
         </div>
-        <div class="background-image">
+        <div class="hero">
             <img src="images/illustration-header-mobile.png" alt="">
         </div>
```

Comment:
- [ ] As requested by the previous reviewer, please remove this part and use CSS `background-image` 👇 property instead of an `img` tag. In the `style.css` file, add this property to the corresponding selector and make sure it looks like the design provided.
```
background-image: url('images/illustration-header-mobile.png');
```